src/runtime.c

```diff
@@ -714,22 +714,28 @@ int main(int argc, char *argv[]) {
 
     int dir_fd, res;
 
-    char mount_dir[64];
+    char temp_base[PATH_MAX] = P_tmpdir;
+    if (getenv("TMPDIR") != NULL)
+      strcpy(temp_base, getenv("TMPDIR"));
+    size_t templen = strlen(temp_base);
+    char mount_dir[templen + 60];
```

Comment:
Where does that 60 come from? Seems a bit like a random guess to me.

Reply:
The original `mount_dir` was defined as `mount_dir[64]`. The `mount_dir` started with `/tmp` and I just looked to keep the suffix size consistent. So, since it was 64 originally with the assumption of `/tmp`, that's where the 60 came from.